cmd/dfget/app/root.go

```diff
@@ -235,6 +235,8 @@ func initFlags() {
 		"identify whether supernode should skip secure verify when interact with the source.")
 	flagSet.IntVar(&cfg.ClientQueueSize, "clientqueue", config.DefaultClientQueueSize,
 		"specify the size of client queue which controls the number of pieces that can be processed simultaneously")
+	flagSet.BoolVar(&cfg.UploadMetrics, "uploadmetrics", true,
```

Comment:
Why do you want to add a flag of `uploadmetrics`? Is there any usage scenario? 
IMOP, monitoring is  something that is not perceived by the users. In addition, the metrics collected by the supernnode will not be complete if we let the user decide whether to report it or not.